internal/cli/atlas_alert_config_fields.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func AtlasAlertConfigsFieldsBuilder() *cobra.Command {
```

Comment:
The ticket was out dated, remember we changed `alert-config` to `alerts config` this should be 
```suggestion
func AtlasAlertsConfigsFieldsBuilder() *cobra.Command {
```
Please also remember to change the file name